quattor/types/aquilon.pan

```diff
@@ -29,10 +29,18 @@ type structure_cluster = {
     "down_hosts_threshold" ? long(0..)
     "node_index" ? long(0..)
 };
- 
+
+type structure_archetype_os = {
+    @{ Name of OS (OperatingSystem.name) }
+    "name" : string
+    @{ Version of OS (OperatingSystem.version) }
```

Comment:
Sorry, yes, it's osversion, it's named OperatingSystem.version in aqdb and aquilon protocols.
I've updated the annotation to hopefully make this clear.